src/Game.tsx

```diff
@@ -363,6 +368,17 @@ function ScoringGame(props: ScoringGameProps) {
     const gameWon = useMemo(() => {
         return playerScores.every((p, i) => i === 0 || !pokerHandLessThan(p.score, playerScores[i - 1].score));
     }, [playerScores]);
+    const { shootConfetti } = useContext(FxContext);
```

Comment:
Why does this need a context and not just a hook?

Reply:
Or rather, it should probably just be a component that is rendered only if the game is not wonnered